Packs/Okta/Playbooks/playbook-Okta_-_User_Investigation.yml

```diff
@@ -1365,6 +1382,213 @@ tasks:
     quietmode: 2
     isoversize: false
     isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 25a5abbc-d30d-4012-8acf-75488d58aed8
+    type: regular
+    task:
+      id: 25a5abbc-d30d-4012-8acf-75488d58aed8
+      version: -1
+      name: Set Number Of Okta Failed Logon to Zero
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      key:
+        simple: NumOfOktaFailedLogon
+      value:
+        simple: "0"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1020,
+          "y": 805
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "41":
+    id: "41"
+    taskid: 13072b32-dac0-4c4d-8c5a-cc0b5de7d1a0
+    type: title
+    task:
+      id: 13072b32-dac0-4c4d-8c5a-cc0b5de7d1a0
+      version: -1
+      name: Failed Logon from ASN Search
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "42"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1470,
+          "y": 265
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "42":
+    id: "42"
+    taskid: 256398af-e248-438b-8e7a-c72f9c238635
+    type: regular
+    task:
+      id: 256398af-e248-438b-8e7a-c72f9c238635
+      version: -1
+      name: Check for all failed logon from ASN
+      description: Gets logs by providing optional filters.
+      script: Okta v2|||okta-get-logs
+      type: regular
+      iscommand: true
+      brand: Okta v2
+    nexttasks:
+      '#none#':
+      - "44"
+    scriptarguments:
+      extend-context:
+        simple: ASNfailedlogs=
+      filter:
+        simple: securityContext.asNumber eq "${inputs.ASN}" and (eventType eq "user.authentication.sso" and outcome.result eq "FAILURE" or eventType eq "user.authentication.auth" and outcome.result eq "FAILURE")
+      ignore-outputs:
+        simple: "true"
+      limit:
+        simple: ${inputs.ASN}
```

Comment:
shouldnt be simple. please change to complex.

Reply:
Done